onnx/defs/experiments/defs.cc

```diff
@@ -363,3 +365,73 @@ OPERATOR_SCHEMA(ATen)
 Experimental allowing ATen operations to be accessed directly from Caffe2
 to allow for quick prototyping when ONNX is missing standard versions of
 and op)DOC");
+
+
+OPERATOR_SCHEMA(ImageScaler)
```

Comment:
Is it equivalent to Scale followed by Add?

Reply:
Yes, but we have one fused OP that does both at once. I put it in experiment with the attention to remove it in the future.